wikicurricula-uy/wikicurricula-boilerplate
# [Feat] : X-Axis changing according to selected sort values

Fixes #28 , #31

Description :  
This pull request is an an enhancement over the above mention PR, 
This PR add the feature on X-axis to change accordingly the selected sort value .

Steps to test :
- Open the wikicurricula-uy main index.html file in browser.
- There by default  article is selected as starting , change that value from the dropdown option. 
- Able to see the visible changes on the x-Axis , X-Axis would change according to the value selected from the dropdown.

Refrence : -


https://github.com/wikicurricula-uy/wikicurricula-boilerplate/assets/97338915/a31914b2-44f8-4f09-93f9-df85d83968fb


Please review it and let me know any changes to do !!
